CHANGELOG.md

```diff
@@ -68,8 +68,11 @@
 
 ### Features
 - Enumerate magic number in RunwayModel [#269](https://github.com/openscope/openscope/issues/269)
+
+- Adds more contex to the Model classes by adding an optional input paramater [#138](https://github.com/openscope/openscope/issues/138)
```

Comment:
make sure this is in the correct spot in the current chagnelog.  this diff looks like its from pre v5.0.0